frontstage/error_handlers.py

```diff
@@ -39,15 +39,17 @@ def connection_error(error):
 
 
 @app.errorhandler(JWTValidationError)
-def jwt_validation_error(error):  # pylint: disable=unused-argument
+def jwt_validation_error(error):
+    del error
     return redirect(url_for('error_bp.not_logged_in_error_page',
                             _external=True,
                             _scheme=getenv('SCHEME', 'http')))
 
 
 @app.errorhandler(Exception)
 def server_error(error):  # pylint: disable=unused-argument
```

Comment:
Not sure but , I think the exception details will get lost in the log using error . The use of error for such checked exception i think would be more appropriate as warning level. They are actually not real errors . 
If you were actually parsing the raising exception here, you would not provably need to change the logging from exception to error. Plus tell if I am wrong but the handler will handle just the one are rised, so no need to change where the exception is not raised.